apps/web-client/src/routes/inventory/inbound/[...id]/+page.svelte

```diff
@@ -124,7 +125,10 @@
 
 	// #region transaction-actions
 	const handleAddTransaction = async (isbn: string) => {
+		const plugin = createBookDataExtensionPlugin();
 		await note.addVolumes({ isbn, quantity: 1 });
+		const book = await plugin.fetchBookData([isbn]);
+		if (book.length) await db.books().upsert(book);
```

Comment:
- Couldn't hurt to add some space between these lines for readability.
- `if` block should be wrapped in `{}`